.drone.yml

```diff
@@ -1,39 +1,62 @@
 branches:
```

Comment:
could you remove `.drone.yml` from the `.prettierignore` file please? I don't remember why we (I?) added this exception, but if we could run it through prettier, it would be nice :)